src/EditorFeatures/TestUtilities/Formatting/AbstractNewDocumentFormattingServiceTests.cs

```diff
@@ -17,33 +17,43 @@ namespace Microsoft.CodeAnalysis.Test.Utilities.Formatting
     public abstract class AbstractNewDocumentFormattingServiceTests
     {
         protected abstract string Language { get; }
-        protected abstract TestWorkspace CreateTestWorkspace(string testCode);
+        protected abstract TestWorkspace CreateTestWorkspace(string testCode, ParseOptions? parseOptions);
 
-        internal Task TestAsync<T>(string testCode, string expected, params (PerLanguageOption2<T>, T)[] options)
+        internal Task TestAsync(string testCode, string expected)
+        {
+            return TestCoreAsync<object>(testCode, expected, options: null, parseOptions: null);
+        }
+
+        internal Task TestAsync<T>(string testCode, string expected, (PerLanguageOption2<T>, T)[]? options = null, ParseOptions? parseOptions = null)
```

Comment:
Is there a test with multiple `(PerLanguageOption2<T>, T)` values?

Reply:
No, there isn't. I could simplify this I guess.

Reply:
Actually, decided to leave it. I don't think it causes much pain, and its reasonable future proofing